ord_schema/orm/README.md

```diff
@@ -94,7 +94,7 @@ methods (this list is not an endorsement of any particular provider):
 # Create a new conda environment.
 conda create -n ord python=3.10
 conda activate ord
-conda install -c rdkit rdkit-postgresql==2020.03.3.0

```

Comment:
I think the version number was there to address #672.

Reply:
The offending SMILES is part of the tests, which are all passing now: https://github.com/open-reaction-database/ord-schema/blob/main/ord_schema/orm/testdata/ord-nielsen-example.pbtxt#L9